apps/aechannel/test/aesc_txs_SUITE.erl

```diff
@@ -825,6 +827,41 @@ slash(Cfg) ->
         ?ROLES),
     ok.
 
+slash_after_lock_timer(Cfg) ->
+    LockPeriod = 10,
+    CloseHeight = 42,
+    Test =
+        fun(Closer, Slasher, SlashHeight) ->
+            run(#{cfg => Cfg, lock_period => 10},
+               [positive(fun create_channel_/2),
+                set_from(Closer),
+                set_prop(round, 10),
+                set_prop(height, CloseHeight),
+                positive(fun close_solo_/2),
+                fun(#{channel_pubkey := ChannelPubKey, state := S} = Props) ->
+                    % make sure the channel is not active any more
+                    ClosedCh = aesc_test_utils:get_channel(ChannelPubKey, S),
+                    false = aesc_channels:is_active(ClosedCh),
+                    Props
+                end,
+                set_from(Slasher),
+                set_prop(round, 20),
+                set_prop(height, SlashHeight),
+                positive(fun slash_/2)])
+        end,
+    lists:foreach(
+        fun(Closer) ->
+            lists:foreach(
+                fun(Slasher) ->
+                    Test(Closer, Slasher, LockPeriod + CloseHeight),
+                    Test(Closer, Slasher, LockPeriod + CloseHeight + 1),
+                    Test(Closer, Slasher, LockPeriod + CloseHeight + 10)
+                end,
+                ?ROLES)
+        end,
+        ?ROLES),
```

Comment:
So this means that one should be able to first solo-close, then follow up with a slash?
(`Test(initiator, initiator, ...)`)

Reply:
yes, indeed